### ISSUE TITLE
custom train-val split and ground planes for test set

### ISSUE THREAD

**Issue by mrshim1101** [kujason/avod#153]
Hi!

First, thank you so much for sharing your work AVOD on Kitti dataset.

I managed to recreate your validation set results and got some trouble for generating similar test results on Kitti. 

I found that @melfm used custom 85-15 split for uploading the test results on Kitti server.

Could you share this split as well as plane data used for test test? 

I really appreciate your help. 

Thanks!

**Comment by kujason** [comment]
#115
